docs/pages/config/database/drivers.mdx

```diff
@@ -76,18 +76,14 @@ database:
       - id: redis-cache
         driver: redis
         redis:
-          addr: YOUR_REDIS_ADDRESS_HERE
-          password: YOUR_REDIS_PASSWORD_HERE
-          db: XXX
-          connPoolSize: 8
+          # Connection URI (Required)
+          # Format: redis://[username]:[password]@[host][:port][/database][?dial_timeout=value1&read_timeout=value2&write_timeout=value3&pool_size=value4]
+          uri: "redis://username:password@host:port/db?dial_timeout=5s&read_timeout=1s&write_timeout=2s&pool_size=10"
           tls:
```

Comment:
How is TLS handled? Will `rediss://` take care of it? What would be effect of having both configured?